prow/config/jobs_test.go

```diff
@@ -95,7 +95,7 @@ func TestPresubmits(t *testing.T) {
 					t.Errorf("Jobs %s and %s have the same context: %s", job.Name, job2.Name, job.Context)
 				}
 				if job2.re.MatchString(job.RerunCommand) {
-					t.Errorf("RerunCommand \"%s\" from job %s matches \"%v\" from job %s but shouldn't.", job.RerunCommand, job.Name, job2.Trigger, job2.Name)
+					t.Errorf("RerunCommand \"%s\" from job %s matches \"%v\" from job %s but shouldn't.", job.RerunCommand, job.Name, job2.Trigger, job2.Name) //TODO
```

Comment:
it was the comment to remind me which part of codes that i need to change. i forgot to remove it afterwards...